src/patterns/api-lambda.test.ts

```diff
@@ -53,4 +56,46 @@ describe("The GuApiLambda pattern", () => {
     });
     expect(stack).toHaveResource("AWS::CloudWatch::Alarm");
   });
+
+  it("should allow us to link a domain name to a LambdaRestApi", () => {
+    const stack = simpleGuStackForTesting();
+    const noMonitoring: NoMonitoring = { noMonitoring: true };
+    const apiLambda = new GuApiLambda(stack, "lambda", {
+      fileName: "my-app.zip",
+      handler: "handler.ts",
+      runtime: Runtime.NODEJS_12_X,
+      monitoringConfiguration: noMonitoring,
+      app: "testing",
+      apis: [
+        {
+          id: "api",
+          description: "this is a test",
+        },
+        {
+          id: "api2",
+          description: "this is a test2",
+        },
+      ],
+    });
+    const api = apiLambda.apis.get("api");
+    expect(api).toBeDefined();
+    /* eslint-disable @typescript-eslint/no-non-null-assertion -- We expect api to be defined beyond this point */
```

Comment:
I'm not a fan of disabling linting to huge blocks as this is also affecting `domainName` on `api`.

Something like this would be preferred:

```ts
const maybeApi = apiLambda.apis.get("api");
expect (api).toBeDefined();

// eslint-disable @typescript-eslint/no-non-null-assertion ......
const api = maybeApi!;
```

As it is more tightly scopes the lint exclusion.

Reply:
That's much clearer, thanks for the suggestion